src/config/mod.rs

```diff
@@ -67,13 +105,127 @@ impl General {
     }
 }
 
+pub fn enabled() -> bool {
+    true
+}
+
+pub fn interval() -> String {
+    "10ms".into()
+}
+
+pub fn distribution_interval() -> String {
+    "50ms".into()
+}
+
+#[derive(Deserialize)]
+pub struct SamplerDefaults {
```

Comment:
Does this need to be its own structure? Couldn't `Config.defaults` be a `SamplerConfig`?